packages/svelte/tsconfig.json

```diff
@@ -14,6 +14,7 @@
 		"strict": true,
 		"allowJs": true,
 		"checkJs": true,
+		"skipLibCheck": true,
```

Comment:
this is probably the bigger change than moving the knip config.  but i guess its fine because svelte itself does not consume libs in a way we would gain something from these checks?

Reply:
yeah, I'm seeing some annoying errors due to some type changes in a dependency that are totally irrelevant to us and to Svelte consumers. I don't see much value in checking libs

Reply:
We can't really do this, we won't be able to catch errors in our own d.ts files from now on. The alternative is to turn all our d.ts files into ts files which _will_ be type checked